src/editor/Renderer.cpp

```diff
@@ -1557,7 +1657,104 @@ void Renderer::setupView() {
 	view.translate(-cameraOrigin.x, -cameraOrigin.z, cameraOrigin.y);
 }
 
+void Renderer::reloadBspModels()
+{
+	isModelsReloading = true;
+
+	if (!mapRenderers.size())
+	{
+		isModelsReloading = false;
+		return;
+	}
+
+	int modelcount = 0;
+
+	for (int i = 0; i < mapRenderers.size(); i++)
+	{
+		if (mapRenderers[i]->map->is_model)
+		{
+			modelcount++;
+		}
+	}
+
+	if (modelcount == mapRenderers.size())
+	{
+		isModelsReloading = false;
+		return;
+	}
+
+	std::vector<BspRenderer*> sorted_renders;
+
+	for (int i = 0; i < mapRenderers.size(); i++)
+	{
+		if (!mapRenderers[i]->map->is_model)
+		{
+			sorted_renders.push_back(mapRenderers[i]);
+		}
+		else
+		{
+			delete mapRenderers[i];
+		}
+	}
+
+	mapRenderers = sorted_renders;
+
+	vector<string> tryPaths = {
+		"./"
+	};
+
+	tryPaths.insert(tryPaths.end(), g_settings.resPaths.begin(), g_settings.resPaths.end());
+
+	for (auto const& bsprend : mapRenderers)
+	{
+		for (auto const& entity : bsprend->map->ents)

```

Comment:
should `is_model` map renderers be skipped?

Reply:
Models add to render list tail, it's needs for something but i don't know for what)))